### PR TITLE
Update msi rollbar version format

### PR DESCRIPTION
**Pull request by daved**
https://www.pivotaltracker.com/story/show/174587605

### CODE REVIEW

File: installers/msi-language/StateCommandExists/CustomAction.cs [hunk 1]
```diff
@@ -9,7 +9,7 @@ public class CustomActions
         [CustomAction]
         public static ActionResult CheckCommands(Session session)
         {
-            ActiveState.RollbarHelper.ConfigureRollbarSingleton(session["COMMIT_ID"]);
+            ActiveState.RollbarHelper.ConfigureRollbarSingleton(session["MSI_VERSION"]);
```
Comment (daved) [inline installers/msi-language/StateCommandExists/CustomAction.cs:12]: I'm calling this as the existing code does, but it doesn't match `session.CustomActionData` in other files. Does this need to be different?

File: scripts/msi-installer-builder/main.go [hunk 2]
```diff
@@ -114,6 +116,11 @@ func normalize(preset languagePreset, c *config) (*config, error) {
 	if err != nil {
 		return c, err
 	}
+
+	if c.MSIVersion == "" {
+		c.MSIVersion = msiVersionInfo()
+	}
```
Comment (Naatan) [inline scripts/msi-installer-builder/main.go:122]: This feels dirty and pointless, we are always setting it to `msiVersionInfo()` one way or another.

**Review by Naatan — CHANGES_REQUESTED** [review]

File: scripts/msi-installer-builder/main.go (hunk 2 above)
Comment (daved) [reply scripts/msi-installer-builder/main.go:122]: The apparent duplication seems less dirty, to me, than the function not doing it's job based on the special knowledge that we happen to set the field in current cases.

Comment (Naatan) [reply scripts/msi-installer-builder/main.go:122]: So error out if it's empty. This is too dirty.

File: installers/msi-language/StateCommandExists/CustomAction.cs (hunk 1 above)
Comment (daved) [reply installers/msi-language/StateCommandExists/CustomAction.cs:12]: @MDrakos or @mdrohmann Would you, please, shed light on why this call is different than other session usages?

Comment (MDrakos) [reply installers/msi-language/StateCommandExists/CustomAction.cs:12]: This custom action is scheduled as `immediate` so it has access to the session object. The custom actions that are deferred do not, so we must pass the data it needs via the `CustomActionData`. This is all done in the `Product.p.wxs` file.

**Review by Naatan — DISMISSED** [review]

**Review by Naatan — APPROVED** [review]

**Pull request merged**